scripts/meta_writer.py

```diff
@@ -78,30 +77,34 @@ async def run(loop, server):
         logger.error("Specified RDB path, %s, does not exist.", args.rdb_path)
         sys.exit(2)
 
-    boto_dict = None
+    botocore_dict = None
     if args.store_s3:
-        boto_dict = katsdpmetawriter.make_boto_dict(args)
-        s3_conn = katsdpmetawriter.get_s3_connection(boto_dict, fail_on_boto=True)
-        if s3_conn:
-            user_id = s3_conn.get_canonical_user_id()
```

Comment:
Is the `user_id` not available anymore in `botocore`, or have you decided against it?

Reply:
The get_canonical_user_id function in boto2 was really just a wrapper around list_buckets that grabbed the user ID from the response.

I've eliminated this check from the script because get_s3_connection already does a call to list_buckets to check the connection.

Reply:
Cool. Was this the last of boto2 in the katsdp codebase? 😂

Reply:
Oh, no doubt there is still something lurking in the archive side of things, and Tom's never-ending quest to move to Python 3 :-)